health-services/project-factory/src/server/app.ts

```
@@ -5,6 +5,8 @@
 import { errorLogger, errorResponder, invalidPathHandler } from './utils/genericUtils';
 import { tracingMiddleware } from './tracing';
 import { createProxyMiddleware } from 'http-proxy-middleware';
+import * as v8 from 'v8';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)